CHANGELOG.md

```diff
@@ -32,6 +32,8 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
   unless repo update was specifically requested.
   [Artem Sheremet](https://github.com/dotdoom)
 
+* Set Default Module for Storyboards in resource bundle targets. [James Treanor](https://github.com/jtreanor)
```

Comment:
please format this properly. Include two spaces after "bundle targets.  "

Reply:
also include an issue or PR number.

Reply:
👍 I think that's correct now.